lib/api/asset/v1/types.ts

```diff
@@ -50,11 +50,18 @@ export interface AssetManagement {
 }
 
 export interface AssetCharacteristics {
-  numberOfBedrooms: number;
-  numberOfLifts: number;
-  numberOfLivingRooms: number;
+  numberOfBedrooms: number | null;
+  numberOfSingleBeds: number | null;
+  numberOfDoubleBeds: number | null;
+  numberOfLifts: number | null;
+  numberOfLivingRooms: number | null;
+  numberOfFloors: number | null;
+  totalBlockFloors: number | null;
+  heating: string;
   windowType: string;
+  propertyFactor: string;
   yearConstructed: string;
+  architecturalType: string;
```

Comment:
I wonder about these... not too familiar with Typescripts types... is the string considered as nullable within typescript? In C# it is, so the question is... if C# API returns "null" for the "heating", will that be an issue on the typescript front-end?

Reply:
Made fields nullable which matches the real data.
```
  heating: string | null;
  windowType: string | null;
  propertyFactor: string | null;
  yearConstructed: string | null;
  architecturalType: string | null;
```